tests/servers/tests.py

```diff
@@ -27,6 +29,34 @@
 }
 
 
+class LiveServerThreadTest(TestCase):
+
+    def run_live_server_thread(self, connections_override=None):
+        thread = LiveServerTestCase._create_server_thread(connections_override)
+
+        thread.daemon = True
+        thread.start()
+        thread.is_ready.wait()
+        thread.terminate()
+        thread.join()
+
+    @unittest.skipIf(connection.vendor == 'sqlite', "sqlite does not permit closing connections")
```

Comment:
I see, I didn't know the SQLite backend `close()` method was already dealing with this case internally. I suggest you only skip the test if an in-memory backend db is used in this case. If you can't do this using `@skipIf` raise a `SkipTest` exception in the test instead.